src/fandom-wirewax.plugin.js

```diff
@@ -0,0 +1,257 @@
+// fandom-jw-plugin.js
+
+console.log("FandomWirewaxPlugin loaded");
+
+function injectEmbedderSDK() {
+  if (window.createWirewaxEmbedder) {
+    console.warn("Embedder SDK is already loaded");
+
+    return;
+  }
+
+  var fandomSDKUrl =
+    "https://edge-assets-wirewax.wikia-services.com/creativeData/sdk-fandom/wirewax-embedder-sdk.js";
+
+  console.log("inject WIREWAX embedder SDK fandom build", fandomSDKUrl);
+
+  return new Promise(function(resolve, reject) {
+    var script = document.createElement("script");
+    script.src = fandomSDKUrl;
+    script.addEventListener("load", resolve);
+    script.addEventListener("error", function(e){reject(e.error)});
+    document.head.appendChild(script);
+  });
+};
+
+function FandomWirewaxPlugin(rootId, options) {
+  this.isPlayerRegistered = false;
+  this.rootId = rootId;
+  this.options = options;
+  this.player = options.player;
+  this.autoPlay = true;
+  this.setupEmbedder = this.setupEmbedder.bind(this);
```

Comment:
All of these functions are defined in the source in fandomSDKUrl for needing to be overridden?

Reply:
These are mostly for use in the embedding/handler functions.